README.md

```diff
@@ -1,51 +1,84 @@
 <div align="center">
-<h1>SnowBall - App Page 🎣</h1>
-
-<p>Snowball combines multiple DeFi protocols to create an interconnected experience. Swap stablecoins, deposit liquidity, or auto-compound liquidity rewards.</p>
+  <h1>Snowball - App Page 🎣 </h1>
+  <p>Snowball combines multiple DeFi protocols to create an interconnected experience. Swap stablecoins, earn by providing liquidity, or auto-compound liquidity rewards through our contracts.</p>
 </div>
 
 ---
 
 ## Summary
 
-This project is the App Page for Snowball and  a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+This project is the App Page for Snowball (front-end) and a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-### Local http://localhost:3000
-### Development https://jupiter-frontend.vercel.app/
+### Local: http://localhost:3000
+### Development: https://jupiter-frontend.vercel.app/
```

Comment:
@Jonasslv Minor issue. Are we planning to change this URL?

Reply:
yeah sure, we can take a look at this later @Ncookiez 
The current development URL is https://app-v2-mocha.vercel.app/